lib/rules/web/admin_console/4.2/common_ui_elements.xyaml

```diff
@@ -110,12 +110,12 @@ click_secondary_menu:
 click_dropdown_field:
   elements:
   - selector:
-      xpath: //label[(.='<dropdown_field>')]/following-sibling::*//button[contains(@class, 'dropdown__toggle')]
+      xpath: //*[(.='<dropdown_field>')]/following-sibling::*//button[contains(@class, 'dropdown__toggle')]
```

Comment:
don't we know type of this element? If we don't, then fine but perhaps better to select the proper element type.

Reply:
Tried to remember why I changed this but with no luck : (, I just remembered that it seems some elements are not in `//label`

Reply:
I remembered why I changed from `//label` to `//*`,as we can see from the following changes in `choose_container`, it use exactly the same xpath expression with `click_dropdown_field`, in that case the element 
```
<div class="co-toolbar__item">Container:</div>
```